pkg/apis/pipeline/v1alpha1/artifact_bucket.go

```diff
@@ -82,14 +82,16 @@ func (b *ArtifactBucket) GetCopyFromContainerSpec(name, sourcePath, destinationP
 	envVars, secretVolumeMount := getSecretEnvVarsAndVolumeMounts("bucket", secretVolumeMountPath, b.Secrets)
 
 	return []corev1.Container{{
-		Name:  names.SimpleNameGenerator.RestrictLengthWithRandomSuffix(fmt.Sprintf("artifact-dest-mkdir-%s", name)),
-		Image: *bashNoopImage,
+		Name:    names.SimpleNameGenerator.RestrictLengthWithRandomSuffix(fmt.Sprintf("artifact-dest-mkdir-%s", name)),
+		Image:   *bashNoopImage,
+		Command: []string{"/ko-app"},
```

Comment:
This will be different depending on the version of `ko` used to produce the images :( 

https://github.com/google/go-containerregistry/pull/380

Reply:
:scream: